test/unit/views/payment-method-view.js

```diff
@@ -145,7 +145,7 @@ describe('PaymentMethodView', () => {
       }
     );
 
-    test('calls model.confirmPaymentMethodDeletion when selection event occurs when in edit mode', () => {
+    test.skip('calls model.confirmPaymentMethodDeletion when selection event occurs when in edit mode', () => {
```

Comment:
I'm not sure what this test needs to accomplish

Reply:
It ensures that the model does delete the payment method when selected, if I recall, via a a keyboard action.